ArduSub/Parameters.h

```diff
@@ -366,7 +369,8 @@ class Parameters {
     AP_Int8         flight_mode6;
     AP_Int8         simple_modes;
     AP_Int8         flight_mode_chan;
-#endif    
+#endif 
```

Comment:
You're changing the whitespace here for no good reason.

If you were *fixing* the problem rather than just changing it I'd probably merge anyway :-)

Reply:
removed the line....line 372 got rid of the trailing whitespace since I had added the index k_param in the wrong spot and removed it and the incorrect whitespace at the end of the #endif got removed automatically...not sure why this is an issue...the added blank line I can understand...